apis/saved.es6.js

```diff
@@ -54,23 +56,41 @@ class Saves extends BaseAPI {
     return super.del(postData);
   }
 
-  save (method, data={}) {
-    // Save, then update the data in the cache (since `saved` model changes also
-    // update `link` or `comment` models)
-    super.save(method, data);
+  updateCache (method, data) {
```

Comment:
I think it would make sense for `updateCache` to be defined in `base` and be called by default when `save` happens, with this logic: https://github.com/schwers/snoode/blob/productionified/apis/base.es6.js#L224